metadata_xml/iso19115-cioos-template/main.j2

```diff
@@ -107,12 +107,10 @@
   </mdb:metadataScope>
   {# contact: mandatory #}
   {% for c in record['contact'] %}
-        {% if not c.inCitation %}
-        {% for role in c.roles %}
+        {% if "custodian" in c.roles %}
```

Comment:
should this be {% if not c.inCitation or "custodian" in c.roles %}?

Reply:
I was thiking that `mdb:contact` would be set to any contact that is set to custodian, regardless of if its also in the citation